dockerfiles/examples/local/README.md

```diff
@@ -0,0 +1,43 @@
+# HPCC Local (host) storage
+
+## Directory contents
+
+### hpcc-localfile/
+
+The hpcc-localfile helm chart will provision a new Persistent Volume and a Persistent Volume Claim for each of the required HPCC storage types.
+Once installed the generated PVC names should be used when installing the HPCC helm chart.
+The values-localfile.yaml is an example of the HPCC storage settings that should be applied, after the "hpcc-localfile" helm chart is installed.
+NB: The output of installing this chart, will contain a generated example with the correct PVC names.
+
+Examples of use:
+
+These all assume that the host directory is /hpccdata (or c:\hpccdata)
+
+#### Docker desktop (using wsl2 to access windows data):
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/hpccdata
+  helm install mycluster hpcc/ --set global.image.version=XXX -f examples/local/values-localfile.yaml
+
+#### Docker desktop (using osx):
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/hpccdata
```

Comment:
Using /hpccdata won't work unless you have added it to the list of directories that Docker shares - by default that list is /Users, /Volumes, /private, /tmp (it seems)

I was using /tmp/hpcc-data in my testing

(note that because /Users is shared, anywhere off home directory should be ok to use, and may be preferable. Not sure non-root users even have access to create /hpccdata)

Reply:
I just tried, and I can't...